lecture-demo/lectureDemo.tex

```diff
@@ -0,0 +1,78 @@
+\documentclass[
+	aspectratio=169, % default is 43
+	8pt, % font size, default is 11pt
+]{beamer}
+
+\usepackage{../beamerthemeuulm} % use the inofficial uulm beamer theme
+
+\title{Beamer Lecture Test}
+\author{Author}
+
+\includeonlylecture{lecture 1}
```

Comment:
would be good to indicate also how to generate the second or all lectures (e.g., with commands in comments)